json-scanf.c

```diff
@@ -55,7 +55,7 @@ struct extractor_specifier {
   bool has_dynamic_size;
   bool allocate_memory;
   bool is_funptr;
-  extractor *funptr;
+  void *funptr;
```

Comment:
this file is not supposed to be changed. Perhaps, you can take a look the last section of the contribution guide line to understand how to properly `rebase` your branch.  It will avoid checking code that should not be changed.